src/box.ts

```diff
@@ -45,46 +56,49 @@ export class BoxPlot extends AVisInstance implements IVisInstance {
   }
 
   get rawSize(): [number, number] {
-    return [300, 50];
+    return [ this.options.width || 300, this.options.heightTo||50];
   }
 
   get node() {
     return <Element>this.$node.node();
   }
 
   private build($parent: d3.Selection<any>) {
-    const size = this.size,
+    const size = [this.options.width,this.options.heightTo],
       data = this.data;
     const $svg = $parent.append('svg').attr({
-      width: size[0],
-      height: size[1],
+      width: this.options.width,
+      height: this.options.heightTo,
       'class': 'phovea-box'
     });
 
-    const $t = $svg.append('g');
+    let height = this.options.width/5>=20?this.options.width/5:20;
```

Comment:
Hard to read. Please use parentheses and format like: `(this.options.width/5 >= 20) ? (this.options.width / 5) : 20;`